drivers/i2c/meson/i2c.c

```diff
@@ -56,38 +56,43 @@ i2c_queue_handle_t queue_handle;
 uintptr_t request_region;
 uintptr_t response_region;
 
-char *meson_token_to_str(uint8_t token) {
+char *meson_token_to_str(uint8_t token)
+{
     switch (token) {
-        case MESON_I2C_TOKEN_END:
-            return "MESON_I2C_TOKEN_END";
-        case MESON_I2C_TOKEN_START:
-            return "MESON_I2C_TOKEN_START";
-        case MESON_I2C_TOKEN_ADDR_WRITE:
-            return "MESON_I2C_TOKEN_ADDR_WRITE";
-        case MESON_I2C_TOKEN_ADDR_READ:
-            return "MESON_I2C_TOKEN_ADDR_READ";
-        case MESON_I2C_TOKEN_DATA:
-            return "MESON_I2C_TOKEN_DATA";
-        case MESON_I2C_TOKEN_DATA_END:
-            return "MESON_I2C_TOKEN_DATA_END";
-        case MESON_I2C_TOKEN_STOP:
-            return "MESON_I2C_TOKEN_STOP";
-        default:
-            return "Unknown token!";
+    case MESON_I2C_TOKEN_END:
+        return "MESON_I2C_TOKEN_END";
+    case MESON_I2C_TOKEN_START:
+        return "MESON_I2C_TOKEN_START";
+    case MESON_I2C_TOKEN_ADDR_WRITE:
+        return "MESON_I2C_TOKEN_ADDR_WRITE";
+    case MESON_I2C_TOKEN_ADDR_READ:
+        return "MESON_I2C_TOKEN_ADDR_READ";
+    case MESON_I2C_TOKEN_DATA:
+        return "MESON_I2C_TOKEN_DATA";
+    case MESON_I2C_TOKEN_DATA_END:
+        return "MESON_I2C_TOKEN_DATA_END";
+    case MESON_I2C_TOKEN_STOP:
+        return "MESON_I2C_TOKEN_STOP";
+    default:
+        return "Unknown token!";
     }
 }
 
-static inline void i2c_dump(volatile struct i2c_regs *regs) {
+/**
+ * Prints the registers of the i2c interface
+ */
+static inline void i2c_dump(volatile struct i2c_regs *regs)
+{
 #ifdef DEBUG_DRIVER
     LOG_DRIVER("dumping interface state...\n");
 
     // Print control register fields
     // uint8_t ctl_man = (ctl & REG_CTRL_MANUAL) ? 1 : 0;
-    uint8_t ctl_rd_cnt = ((regs->ctl & REG_CTRL_RD_CNT) >> 8);
-    uint8_t ctl_curr_tk = ((regs->ctl & REG_CTRL_CURR_TK) >> 4);
-    uint8_t ctl_err = (regs->ctl & REG_CTRL_ERROR) ? 1 : 0;
-    uint8_t ctl_status = (regs->ctl & REG_CTRL_STATUS) ? 1 : 0;
-    uint8_t ctl_start = (regs->ctl & REG_CTRL_START) ? 1 : 0;
+    uint8_t ctl_rd_cnt = ((regs->ctl &REG_CTRL_RD_CNT) >> 8);
+    uint8_t ctl_curr_tk = ((regs->ctl &REG_CTRL_CURR_TK) >> 4);
+    uint8_t ctl_err = (regs->ctl &REG_CTRL_ERROR) ? 1 : 0;
+    uint8_t ctl_status = (regs->ctl &REG_CTRL_STATUS) ? 1 : 0;
+    uint8_t ctl_start = (regs->ctl &REG_CTRL_START) ? 1 : 0;
```

Comment:
There's an issue with the style cehcker we use that's introducing changes like this. Do you mind manually reverting this style change?

Reply:
done